addon/components/olv-toolbar.js

```
@@ -14,6 +14,9 @@ import { getOwner } from '@ember/application';
 import { A } from '@ember/array';
 import FlexberryBaseComponent from './flexberry-base-component';
 import serializeSortingParam from '../utils/serialize-sorting-param';
+import { getNewRouteNameOfEditForm } from '../utils/routes-names-builder';
+import { translationMacro as t } from 'ember-i18n';
+const { getOwner } = Ember;
```

Biome

[error] Shouldn't redeclare 'getOwner'. Consider to delete it or rename it.

'getOwner' is defined here:


(lint/suspicious/noRedeclare)